content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -180,7 +180,7 @@ Kubernetes provides built-in signers that each have a well-known `signerName`:
    1. Permitted subjects - organizations are exactly `["system:nodes"]`, common name starts with "`system:node:`".
    1. Permitted x509 extensions - honors key usage and DNSName/IPAddress subjectAltName extensions, forbids EmailAddress and
       URI subjectAltName extensions, drops other extensions. At least one DNS or IP subjectAltName must be present.
-   1. Permitted key usages - exactly `["key encipherment", "digital signature", "server auth"]`.
+   1. Permitted key usages - `["key encipherment", "digital signature", "client auth"]` if SignatureAlgorithm is RSA; `["digital signature", "client auth"]` otherwise.
```

Comment:
either usage is permitted, it's up to the caller to request the subset if it wants to when using RSA

```suggestion
   1. Permitted key usages - `["key encipherment", "digital signature", "client auth"]` or `["digital signature", "client auth"]`
```

Reply:
updated to recommendation